### ISSUE TITLE
pre-trained weights for YOLOv7

### ISSUE THREAD

**Issue by ohjho** [JackWoo0831/Yolov7-tracker#112]
are you able to share pre-trained weights for the MOT or VisDrone dataset?

**Comment by JackWoo0831** [comment]
Of course! You can try this checkpoint: [https://pan.baidu.com/s/1m13Q8Lx_hrPVFZI6lLDrWQ](https://pan.baidu.com/s/1m13Q8Lx_hrPVFZI6lLDrWQ), the code is: ndkf
